### DISCUSSION TITLE
Method for changing backlight settings

### DISCUSSION THREAD

**Discussion by Beoge** [MatrixEditor/fsapi-tools#16] [General]
Hi,

I have been looking for a way to change the **backlight settings** remotely via FSAPI for some time now.
In fsapi-tools/nodes/nodes.txt the following nodes are  specified:

- io.display.backlight
- io.display.passivebacklight
- io.display.screen

Can these be changed via API?

The background to this is that on my radio device (with VENICE X / FS2340) only the backlight can be set in the system settings during operation, but not in standby mode. My hypothesis is that the chip could actually change the brightness in standby mode with software like other radios on the same basis, but I would have to change something at a technically lower level (for example via FSAPI call).

**Comment by MatrixEditor** [comment]
Hi Beoge!

Yes, you should be able to modify the backlight settings of your device (**if** supported) during standby mode using HTTP-Requests. The available nodes were taken from decompressed firmware files and most of them are not verified. Therefore, it is possible that your radio doesn't even support the mentioned nodes. Otherwise, you would need do reverse engineer how to adjust/modify the web-request parameters.
The web-service of a radio should be active as long as it hat a power source. 

I would suggest to first test whether the radio supports the mentioned nodes using HTTP Requests.
